src/rules/enums.ts

```diff
@@ -1,12 +1,58 @@
+import { AST_NODE_TYPES } from "@typescript-eslint/utils";
+
 import { createRule } from "../utils.js";
 
 export const rule = createRule({
 	create(context) {
 		return {
 			TSEnumDeclaration(node) {
+				const name = node.id.name;
+				let isExported = false;
+				if (node.parent.type === AST_NODE_TYPES.ExportNamedDeclaration) {
+					isExported = true;
+				}
+				let count = 0;
+				let body = ``;
+				for (const enumMember of node.body.members) {
+					if (enumMember.id.type !== AST_NODE_TYPES.Identifier) {
+						return;
+					}
+					const propertyName = enumMember.id.name;
+					if (
+						enumMember.initializer &&
+						enumMember.initializer.type === AST_NODE_TYPES.Literal
```

Comment:
[Style] Nit, I'm surprised this wasn't flagged by linting:

```suggestion
						enumMember.initializer?.type === AST_NODE_TYPES.Literal
```